### ISSUE TITLE
Chat between users not working

### ISSUE THREAD

**Issue by canove** [Ylianst/MeshCentral#740]
Hi everyone! I just found and started using MeshCentral a few days ago, and I'm impressed! Its amazing tool. Thank you very much for this!

In my fresh install, the chat between User and Remote Computer are working well, but chat between users, don't. Every time I click on chat icon, it brings me to the users profile and no chat pup's up.

Do I need to enable this function somewhere? 

Thanks in advance!

I have another question, and I don't know if i should open another issue for this:

When trying to brand MeshCentral with my company logo.
I just can't set a title to MeshCentral webpage AND a title-picture. If I do, the title text will appear in same place as my company logo. I prefer logo instead of text title, but in this case the webpage "name" on browser will show my ip address.

**Comment by Ylianst** [comment]
First, thanks for the feedback.
I just tested it and your right on the user chat not working. I am working on a fix now.
For the title-picture/title... your are also right.
Both small issues. I will try to have both fixed in the next release.

**Comment by Ylianst** [comment]
I have a fix for the user chat, it got broken when internationalizing the web site. When I publish the next version, it will be fixed. I am also fixing the title stuff and will have new instructions for that tomorrow.

**Comment by Ylianst** [comment]
Published MeshCentral v0.4.5-y with fixed for both issues above. User chat now works again and for the title stuff, it now works better:

```
"_titlehtml": "<div style=\"height:66px;color:#c8c8c8;padding-left:14px;padding-top:8px;font-size:46px;font-family:Arial,Helvetica,sans-serif;font-weight:bold\">This is a test</div>",
"title": "MySuperServer",
"_title1": "MyServer",
"_title2": "ABC",
"_titlePicture": "title-test.png"
```

You can play with these to see what they do. "title1" is new, but only used to force the text to show even with HTML or a Picture is shown, lot likely to be used a lot.

**Comment by canove** [comment]
Both user chat and html title works great. Thanks!
